### ISSUE TITLE
acc on imagnet

### ISSUE THREAD

**Issue by WenmuZhou** [DefangChen/SimKD#5]
In the document, why the teacher model trained on the imagenet dataset is resnet18, when training the student model, the teacher model is resnet50, and the student model is resnet18. In addition, can you tell me the acc on imagnet?

**Comment by DefangChen** [comment]
All teacher models and vanilla student models are trained with [train_teacher.py](https://github.com/DefangChen/SimKD/blob/main/train_teacher.py).

The line 39-40 of [scripts](https://github.com/DefangChen/SimKD/blob/main/scripts/run_vanilla.sh) actually train a vanilla student model (resnet18) in ImageNet dataset. You can use a simply modified version of this command to train resnet50 or use our provided pre-trained model.

**Comment by WenmuZhou** [comment]
So in Table 3 in the paper, the teacher model is resnet50, the student model is resnet18, and the resnet18 accuracy obtained by distillation is 71.66%. Can I use the code in the repo to achieve this accuracy?
![image](https://user-images.githubusercontent.com/12406017/180180103-16ea9709-7c87-4d97-97dd-109561891b23.png)

**Comment by DefangChen** [comment]
Yes.
